code/_onclick/hud/rendering/plane_master_group.dm

```diff
@@ -13,11 +13,15 @@
 	var/active_offset = 0
 	/// What, if any, submap we render onto
 	var/map = ""
+	/// Due to a Byond bug where secondary maps have improperly offset render targets, we will need to override this to "LEFT,TOP" for clients >515.1614
+	var/relay_loc = "CENTER"
 
-/datum/plane_master_group/New(key, map = "")
+/datum/plane_master_group/New(key, map = "", relay_loc)
 	. = ..()
 	src.key = key
 	src.map = map
+	if(relay_loc)
+		src.relay_loc = relay_loc
```

Comment:
there's no need to make this an argument, it's a simple change so it's better to just let people make this change themselves if they want to

Reply:
removed it